spec/models/post_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.create(name: 'Henry', photo: 'https://www.coolpicturehostingsite.com/', bio: 'Pretty cool guy!')
+
+  subject { Post.new(user:, title: 'Cool title', text: 'Sweet text') }
+
+  before { subject.save }
+
+  it 'Post should be valid' do
+    expect(subject).to be_valid
+  end
+
+  it 'title should be under 250 characters in length' do
+    expect(subject.title.size).to be < 250
+  end
+
+  it ':recent_comments should return 5 posts even if there are more posts to gather' do
+    Comment.create(text: 'text for post 1', user:, post: subject)
+    Comment.create(text: 'text for post 2', user:, post: subject)
+    Comment.create(text: 'text for post 3', user:, post: subject)
+    Comment.create(text: 'text for post 4', user:, post: subject)
+    Comment.create(text: 'text for post 5', user:, post: subject)
+    Comment.create(text: 'text for post 6', user:, post: subject)
+    expect(subject.recent_comments.size).to eq(5)
+  end
+
+  it 'expect post counter to increment' do
```

Comment:
- [ ] Please fix these failing tests
![spec](https://user-images.githubusercontent.com/44745855/178071508-df55efa3-f520-4c65-aaac-687d12f18dae.png)

Reply:
@DeliceLydia Hi! on my PC my tests pass. I have made no changes and this is my output:

![image](https://user-images.githubusercontent.com/92178070/178074484-2980f0e6-d327-4957-92d4-9823f63d52e2.png)

Are you able to show me the details of the errors you are receiving?

Reply:
@DeliceLydia 

I had another student download the repo and run the tests. They had to follow these steps

- `git checkout validation_and_specs`
- `bundle install`
- `rails db:create`
- `rspec`

Otherwise, maybe a package was missing or something since all the installs weren't completed beforehand. The tests passed as well.

Can you try this and let me know if this doesn't work? The logic should be passing, but I think that maybe the setup on your PC wasn't complete